### DISCUSSION TITLE
Welcome to parsera Discussions!

### DISCUSSION THREAD

**Discussion by raznem** [raznem/parsera#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bin-blaster** [comment]
What should I do if the returned results are incorrect?

**Reply by DrWebDeveloper** [reply]
I think you should try to revise the description

**Reply by raznem** [reply]
Could you provide more context on what exactly is incorrect?

**Reply by bin-blaster** [reply]
I tried with the following parameters:
url = "https://e.vnexpress.net/news/life/trend/foreign-students-find-much-love-from-vietnamese-mothers-4782246.html"
elements = {
    "Title": "News title",
    "Summary": "Summary of the news",
    "Content": "Content of the news",
}
The returned result: 
[{'Title': 'Foreign students find much love from Vietnamese mothers', 'Summary': 'Phuong Anh cooked a meal for seven people, including her three children and four Laos students in HCMC whom she considered her own.', 'Content': 'For lunch one weekend Phuong Anh cooked a meal for seven people, including her three children and four Laos students in HCMC whom she considered her own. The resident of Tan Binh District made fried spring rolls and the children assisted her with picking vegetables and chopping garlic and chilli. The meal featured Vietnamese and Laos dishes, including lap (Laos meat salad), bun cha (Vietnamese grilled pork with rice noodles) and the spring rolls. To cater to her guests’ palate, Phuong Anh she made the seasoning spicy and salty. After the meal they spent a few hours chatting. Anh then packed some fruits for the students to take back to their dormitory in District 3.'}]
The returned Title and Summary are correct, but the returned Content field returns incomplete content.

**Reply by raznem** [reply]
I see, existing extractor is the best suited for short outputs, like summary and title. Current version wouldn't work well with longer content. For extraction of articles I used Mozilla's readability code https://github.com/mozilla/readability (this is the code they use in the browser for reading mode for pages), so maybe check it out.
Parsera will be capable of handling that only when we'll manage to develop alternative extraction flow, that uses LLM for scraping code generation instead of content extraction.
